src/interfaces/store-interfaces/state-interface.ts

```diff
@@ -64,6 +65,7 @@ export interface StateModelIF {
   feePrices: Array<FeesIF>
   registration: RegistrationStateIF
   completingParty?: CompletingPartyIF
+  parties?:Array<PartyIF>
```

Comment:
We usually have a space after the `:`.

`npm run lint` might fix fix automatically.